doc/themes/the-things-stack/layouts/shortcodes/cluster-selector.html

```diff
@@ -4,6 +4,7 @@
     <div class="select">
       <select data-content="cluster-id">
         <option value="eu1" selected>Europe 1 (Ireland)</option>
+        <option value="eu2">Europe 2 (UK)</option>
```

Comment:
```suggestion
```

Only The Things Stack Cloud has an `eu2` cluster. The Things Stack Community Edition doesn't.